package.json

```diff
@@ -55,7 +55,7 @@
     "test:performance:server": "serve test/performance -p 5001",
     "test:argos": "node ./scripts/pushArgos.mjs",
     "typescript": "lerna run --no-bail --parallel typescript",
-    "typescript:ci": "lerna run --concurrency 5 --no-bail --no-sort typescript",
+    "typescript:ci": "lerna run --concurrency 4 --no-bail --no-sort typescript",
```

Comment:
Done a similar change as in https://github.com/mui/mui-x/pull/6850 to reduce the RAM usage and potentially avoid overflowing the available amount.
**Before:**
https://app.circleci.com/pipelines/github/mui/mui-x/33679/workflows/0eafce89-b995-4469-b6bd-8fe65956b51a/jobs/193177/resources
**After:**
https://app.circleci.com/pipelines/github/mui/mui-x/33681/workflows/3ad24b9a-5f7c-4681-84f3-8f1c5c396b39/jobs/193182/resources

Notice that we don't have a 99% and following steady usage decline signaling that the script probably exited.

Reply:
Agreed, but for now,
![Screenshot 2023-02-14 at 13 01 04](https://user-images.githubusercontent.com/4941090/218717193-7f5fa4ae-15bd-486d-809f-faa8613fa260.png)
 it does not seem to differ basically at all. 🤔

Reply:
By the way, MUI Core (material-ui) repo is using "Medium+" container, which has 3 CPU cores (up from 2) and 6GB of RAM (up from 4GB).

Reply:
If MUI Core has a better container, maybe we could consider having it as well